src/journal/journald-context.c

```diff
@@ -179,6 +180,9 @@ static void client_context_reset(Server *s, ClientContext *c) {
 
         c->log_ratelimit_interval = s->ratelimit_interval;
         c->log_ratelimit_burst = s->ratelimit_burst;
+
+        c->log_filter_regex = pattern_free(c->log_filter_regex);
+        c->log_filter_deny_list = false;
```

Comment:
Not necessary to set `false`, I think, as the boolean flag will not be used if `log_filter_regex` is null.